domain/src/main/java/org/oppia/android/domain/oppialogger/OppiaLogger.kt

```diff
@@ -14,19 +13,17 @@ class OppiaLogger @Inject constructor(
   /** Logs transition events. See [AnalyticsController.logTransitionEvent] for more context. */
   fun logTransitionEvent(
     timestamp: Long,
-    eventAction: EventAction,
     eventContext: EventLog.Context?
```

Comment:
Since you removed the tests checking for nullness, should a non-null context now be required here & elsewhere?

Reply:
done.